go.mod

```diff
@@ -52,7 +52,7 @@ require (
 	github.com/prometheus/alertmanager v0.26.0
 	github.com/prometheus/client_golang v1.18.0
 	github.com/prometheus/client_model v0.5.0
-	github.com/prometheus/common v0.46.0
+	github.com/prometheus/common v0.46.1-0.20240206181200-773d5664eb8d
```

Comment:
Let's not pin to specific commits here. I have just tagged 0.47.0 in common, so you can use it now.

Reply:
I don't see the tag on github, do you need to push it?

Reply:
I have pushed it an hour ago, and even created a nice release.

Maybe Github is once more very eventually consistent?
https://github.com/prometheus/common/releases/tag/0.47.0